File: redsheet-portal/src/app/database-models/v-lite-target-record.ts
import { required, maxLength, range, nested } from '@rx/annotations';

export class vLiteTargetRecord {
    constructor(vLiteTargetRecord?: vLiteTargetRecord )  {
        let properties = [ "liteProjectBackgroundId", "liteTargetDetail", "liteTargetId",];
        for (let property of properties)
            if (vLiteTargetRecord && vLiteTargetRecord[property])
                this[property] = vLiteTargetRecord[property];
    }
 
	liteProjectBackgroundId : number =   0 ;
 
    @required()
    @maxLength(50)
	liteTargetDetail : string =   undefined;
 
    @range(1,2147483647)
	liteTargetId : number =   undefined;


}

File: RedSheet.DbEntities/Models/vLiteTargetRecord.cs

using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.ComponentModel.DataAnnotations;
using Microsoft.EntityFrameworkCore;
using Rx.Core.Data.Attributes;
using RedSheet.DbEntities.Enums;

namespace RedSheet.DbEntities.Models
{
	[Table("vLiteTargetRecords", Schema = "dbo")]
	[Serializable]
	public partial class vLiteTargetRecord  
	{
 
	[DatabaseGenerated(DatabaseGeneratedOption.None)] 
	[Key]
	public int LiteProjectBackgroundId { get; set; } 
 
	[Required] 
	[MaxLength(50)]
	public string LiteTargetDetail { get; set; } 
 
	[DatabaseGenerated(DatabaseGeneratedOption.Identity)] 
	[Range(1, int.MaxValue)]
	public int LiteTargetId { get; set; } 

	public vLiteTargetRecord()
	{
    }

    }
}

